.kitchen.yml

```diff
@@ -14,24 +14,34 @@ provisioner:
   name: chef_zero
 
 platforms:
-  - name: windows-8.1-chef11.10.4
+  # - name: windows_2008r2-chef11
```

Comment:
To make it possible to add it back in again once we have a vagrantbox that works again. But I'm cool with whacking it. I'll do so now.